mdformat/renderer/_token_renderers.py

```diff
@@ -101,9 +104,10 @@ def fence(tokens: List[Token], idx: int, options: dict, env: dict) -> str:
         fmt_func = options["codeformatters"][lang]
         try:
             code_block = fmt_func(code_block, info_str)
-        except Exception:
+        except Exception as err:
             # Swallow exceptions so that formatter errors (e.g. due to
             # invalid code) do not crash mdformat.
+            LOGGER.warning(f"Code formatting of '{lang}' failed: {err}")
             pass
```

Comment:
Remove this line?

Reply:
If we are addressing later it in #37 then yeh sure

Reply:
I refer to line 111, not 110 :smile:

Reply:
oh yeh oops

Reply:
we can probably improve on this anyway, but will leave it in there for a start